Marlin/Marlin_main.cpp

```diff
@@ -5688,11 +5688,18 @@ inline void gcode_M77() { print_job_timer.stop(); }
 /**
  * M104: Set hot end temperature
  */
+#if ENABLED(SINGLENOZZLE) || ENABLED(SINGLENOZZLE_SWITCHING_EXTRUDER)
```

Comment:
i don't know, that i understand you right and i hope i changed it correct.i don't know, i understand you right and change it correct.